core/storage/base_model/gae_models.py

```diff
@@ -161,6 +179,26 @@ def get_new_id(cls, entity_name):
     @classmethod
     def _fetch_page_sorted_by_last_updated(
             cls, query, page_size, urlsafe_start_cursor):
+        """Fetches page of entities sorted by last updated.
```

Comment:
page --> a page
last updated --> their last_updated attribute.

Sorted in descending or ascending order? Should also specify.